contrib/helm/harbor/values.yaml

```diff
@@ -301,17 +302,30 @@ postgresql:
   persistence:
     enabled: false
 
+## Settings for redis dependency.
+## see https://github.com/kubernetes/charts/tree/master/stable/redis
+## for further configurables.
+redis:
+# Update needed in the cm that defines redis_url if usePassword is set to true.
+  usePassword: false
+  cluster:
+    enabled: false
+  master:
+    persistence:
+# temp workaround the permission issue.
```

Comment:
suggestion add a TODO here, and explain the details for permission issue (e.g. a bug url)